src/styles/blocks/stars.css

```diff
@@ -0,0 +1,39 @@
+.stars__card {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 16px 24px;
+}
+
+.stars__list {
+  display: flex;
+}
+
+.stars__item:not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars__item--one,
+.stars__item--two,
+.stars__item--three,
+.stars__item--four {
+  height: 16px;
+  width: 16px;
+  background-image: url(../../images/star-active.svg);
+  background-repeat: no-repeat;
+}
```

Comment:
You don't need to add modifiers to stars. Use pseudo-classes instead. `:last-child` for example